Codemine.xcodeproj/xcshareddata/xcschemes/Codemine.xcscheme

```diff
@@ -26,8 +26,8 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "YES"
-      codeCoverageEnabled = "NO">
+      codeCoverageEnabled = "YES"
+      shouldUseLaunchSchemeArgsEnv = "YES">
```

Comment:
Are these changes by accident?

Reply:
I showed him how to enabled code coverage when unit testing

Reply:
I showed him how to enable code coverage. One could argue if it should be part of the entire framework or just set up locally